### PR TITLE
[RNMobile] Update icon for UngroupButton component

### PR DESCRIPTION
**Pull request by t-hamano**
Part of #50520
Related to #17575

## What?
This PR updates the ungroup icon that appears on the toolbar at the bottom of the screen.

ungroup

## Why?
This icon was hard-coded SVG. However, an icon representing the same meaning already exists in the `icons` package, so I thought it made sense to use it.

## Testing Instructions

I am a Windows user and tried to debug using Android Studio, but I just could not get the emulator to work properly.

The following icon should appear instead:

new-ungroup

I apologize for not being able to test this, but I would appreciate it if you could check to see if this change makes sense.

### CODE REVIEW

File: packages/block-editor/src/components/ungroup-button/index.native.js [hunk 1]
```diff
@@ -6,11 +6,11 @@ import { ToolbarGroup, ToolbarButton } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { withSelect, withDispatch } from '@wordpress/data';
 import { compose } from '@wordpress/compose';
+import { Ungroup } from '@wordpress/icons';
```
Comment (fluiddot) [inline packages/block-editor/src/components/ungroup-button/index.native.js]: @t-hamano I noticed when testing that the icon wasn't being rendered. I checked [the icons index file](https://github.com/WordPress/gutenberg/blob/bc7030daa86543d6d9301bb9844caa09de36c4b8/packages/icons/src/index.js#LL258C21-L258C28) and all are exported in lowercase, hence we'd need to update the import statement:
```suggestion
import { ungroup } from '@wordpress/icons';
```

**Review by fluiddot — CHANGES_REQUESTED** [review]

File: packages/block-editor/src/components/ungroup-button/index.native.js (hunk 1 above)
Comment (t-hamano) [reply packages/block-editor/src/components/ungroup-button/index.native.js]: @fluiddot 
Thank you for testing it. I fixed it with ce87ef7.

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 ! Thanks @t-hamano for updating this 🙇 !

I confirmed the Ungroup icon is displayed by using the Group block. Note that the icon changed:

| Before | After |
|--------|--------|
|  |  | 

_Tested on Simulator - iPhone 14 (iOS 16.0)_

**Comment by t-hamano** [comment]
Thanks for the review, @fluiddot!

**Pull request merged**